pythonfmu/tests/test_variables.py

```diff
@@ -35,6 +37,64 @@ def test_ScalarVariable_constructor(causality, initial, variability, name, descr
     assert var.variability == variability
 
 
+@pytest.mark.parametrize("fmi_type,value", [
+    (Boolean, False), 
+    (Integer, 22), 
+    (Real, 2./3.), 
+    (String, "hello_world"),
+])
+def test_ScalarVariable_getter(fmi_type, value):
+    
+    class Slave(Fmi2Slave):
+
+        def __init__(self, **kwargs):
+            super().__init__(**kwargs)
+            self.var = [value, ]
```

Comment:
What's the reason for declaring arrays in this way `, ]`

Reply:
definitely unneeded